getting-started/server-blazor.md

```diff
@@ -1,7 +1,8 @@
 ---
 title: Blazor Server App
 page_title: First Steps with Blazor Server Apps and Telerik UI for Blazor
-description: Make your first steps with Telerik UI for Blazor and build an app that runs the UI components server-side in Blazor server.
+meta_title: First Steps with Blazor Server-Side UI - Telerik UI for Blazor
+description: Blazor Server is a .NET framework that runs C# on the server, enabling interactive UIs via SignalR.
```

Comment:
Here the SEO goal has overruled the sensible description of the article, which no longer explains what the article is about. Instead, the description focuses on general knowledge for the sake of keywords.